### PR TITLE
Initial Codemods

### PR DESCRIPTION
**Pull request by lemonmade**
This PR adds an initial Codemod (to do the Mocha context -> closure conversion), as well as some basic tooling to make it easier to test other transforms. I also did some minor housekeeping in other parts of the project.

cc/ @bouk

### CODE REVIEW

File: packages/shopify-codemods/src/mocha-context-to-closure.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
+import takeWhile from 'lodash/takeWhile';
```
Comment (bouk) [inline packages/shopify-codemods/src/mocha-context-to-closure.js]: You don't need to compile codemods, as jscodeshift runs them with babel registered

File: packages/shopify-codemods/src/mocha-context-to-closure.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    ));
+
+    return path.node;
+  }
+
+  return root
+    .find(j.CallExpression)
+    .filter((contextPath) => contextPath.scope.isGlobal && createsContext(contextPath))
+    .replaceWith((contextPath) => handleContext(contextPath))
+    .toSource({quote: 'single'});
```
Comment (bouk) [inline packages/shopify-codemods/src/mocha-context-to-closure.js]: Please accept printOptions, like js-codemod https://github.com/cpojer/js-codemod/blob/37ba134f9eb038571c0e48590c672ee4b531f1bd/transforms/object-shorthand.js#L66

File: packages/shopify-codemods/src/mocha-context-to-closure.js (hunk 1 above)
Comment (bouk) [inline packages/shopify-codemods/src/mocha-context-to-closure.js]: Also, why pull in lodash for this single function? Just inline itg

**Comment by bouk** [comment]
The package should probably be named `shopify-codemod` (no s) as there's `react-codemod`, `js-codemod` etc.

File: packages/shopify-codemods/src/mocha-context-to-closure.js (hunk 1 above)
Comment (lemonmade) [inline packages/shopify-codemods/src/mocha-context-to-closure.js]: Yeah I fucked up the compile stuff anyways, lol

Comment (lemonmade) [inline packages/shopify-codemods/src/mocha-context-to-closure.js]: And inlining sounds good :+1:

**Pull request merged**